iroha_data_model/src/lib.rs

```diff
@@ -1455,11 +1463,68 @@ pub mod transaction {
         }
     }
 
+    #[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode, PartialEq, Eq)]
+    pub enum FinalizedTransaction {
+        Accepted {
+            transaction: Transaction,
```

Comment:
Our styleguide forbids such enums. You should wrap Accepted and Rejected variants into separate structures.

https://github.com/hyperledger/iroha/blob/iroha2-dev/CONTRIBUTING.md#rust-style-guide

Reply:
Agree